common/djangoapps/request_cache/__init__.py

```diff
@@ -73,11 +73,6 @@ def get_request_or_stub():
     request = crum.get_current_request()
 
     if request is None:
-        log.warning(
```

Comment:
Great question. I should have mentioned that @ssemenova and I discovered yesterday that teams has a management command to regenerate its search index, and this also seems not to be executed in a request context:

https://github.com/edx/edx-platform/blob/master/lms/djangoapps/teams/search_indexes.py#L52

We talked about whether we could refactor this, but we decided that it was simpler to just declare that this method is supported.